src/Shared/Recruit.Vacancies.Client/Application/Rules/VacancyRules/VacancyRuleSet.cs

```diff
@@ -1,18 +1,22 @@
-﻿using Esfa.Recruit.Vacancies.Client.Application.Providers;
+﻿using Esfa.Recruit.Vacancies.Client.Application.Configuration;
+using Esfa.Recruit.Vacancies.Client.Application.Providers;
 using Esfa.Recruit.Vacancies.Client.Application.Rules.Engine;
+using Esfa.Recruit.Vacancies.Client.Application.Services;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 
 namespace Esfa.Recruit.Vacancies.Client.Application.Rules.VacancyRules
 {
-    public class VacancyRuleSet: RuleSet<Vacancy>
-    {        
-        public VacancyRuleSet(
-            IProfanityListProvider profanityListProvider,
-            IBannedPhrasesProvider bannedPhrasesProvider) 
-            : base(nameof(VacancyRuleSet))
+    public class VacancyRuleSet : RuleSet<Vacancy>
+    {
+        public VacancyRuleSet(QaRulesConfiguration qaRulesConfig,
+                    IApprenticeshipProgrammeProvider apprenticeshipProgrammeProvider,
```

Comment:
The intent of this is because qaRulesConfig will have more going on and can be used in multiple rules which we are adding in this ctor. Not against dependency injecting rules whoever should we do that once we have more rules.